config/initializers/mappings/publish_to_allocation_subject_mapping.rb

```diff
@@ -28,6 +46,17 @@ module PublishSubjects
   PublishSubjects::SPANISH,
 ].freeze
 
+PUBLISH_PRIMARY_SUBJECT_SPECIALISM_MAPPING = {
+  # Primary subject and its derivatives
+  PublishSubjects::PRIMARY => [CourseSubjects::PRIMARY_TEACHING],
+  PublishSubjects::PRIMARY_WITH_ENGLISH => [CourseSubjects::PRIMARY_TEACHING, CourseSubjects::ENGLISH_STUDIES],
+  PublishSubjects::PRIMARY_WITH_GEOGRAPHY_AND_HISTORY => [CourseSubjects::PRIMARY_TEACHING, CourseSubjects::GEOGRAPHY, CourseSubjects::HISTORY],
+  PublishSubjects::PRIMARY_WITH_MATHEMATICS => [CourseSubjects::SPECIALIST_TEACHING_PRIMARY_WITH_MATHEMETICS],
+  PublishSubjects::PRIMARY_WITH_MODERN_LANGUAGES => [CourseSubjects::PRIMARY_TEACHING, CourseSubjects::MODERN_LANGUAGES],
+  PublishSubjects::PRIMARY_WITH_PHYSICAL_EDUCATION => [CourseSubjects::PRIMARY_TEACHING, CourseSubjects::PHYSICAL_EDUCATION],
+  PublishSubjects::PRIMARY_WITH_SCIENCE => [CourseSubjects::PRIMARY_TEACHING, CourseSubjects::GENERAL_SCIENCES],
```

Comment:
Note that the subject has changed from 
`[CourseSubjects::PRIMARY_TEACHING, CourseSubjects::SCIENCE]`
`[CourseSubjects::PRIMARY_TEACHING, CourseSubjects::GENERAL_SCIENCES]`
following https://github.com/DFE-Digital/register-trainee-teachers/pull/1414/commits/49e545e3bc109161f15efe431cb07e23b6989a1f on https://github.com/DFE-Digital/register-trainee-teachers/pull/1414